active/events-support.md

```diff
@@ -0,0 +1,54 @@
+# Events Support Working Group
+
+## Scope of responsibilities
+
+The group supports organizers of Django-related events globally, including DjangoCon conferences. The group acts on behalf of the DSF Board to:
+
+- Be the point of contact for organizers to ask questions and seek guidance.
+- Proactively provide support to event organizers.
+- Debrief with organizers after events.
+- Maintain shared resources to facilitate continuity and knowledge transfer between events.
+- Actively promote diversity and inclusion in event support practices, encouraging organizers to create inclusive events that welcome and represent diverse communities.
+
+For DjangoCon events, the working group will:
+
+- Liaise with DEFNA (Django Events Foundation North America) who oversee DjangoCon US.
+- Manage and execute the Call for Organizers process for DjangoCon Europe, providing recommendations to the DSF board for final approval.
+- Assist DjangoCon Europe organizers with event planning, logistics, and execution.
+- Work with DjangoCon Africa organizers to make this event sustainable.
```

Comment:
```suggestion
- Work with DjangoCon Africa organizers to make this event sustainable.
- Work with the Social Media Working Group to promote events
```
Would it make sense?

Reply:
I think so! Not sure in what way, where the remit of the social media group is and what’s for this events group.